spec/system/support_interface/docs_spec.rb

```diff
@@ -32,6 +32,7 @@ def and_it_contains_documentation_for_all_emails
       candidate_mailer-survey_chaser_email
       candidate_mailer-survey_email
       provider_mailer-account_created
+      provider_mailer-application_withrawn
```

Comment:
This should be added in these places:

https://github.com/DFE-Digital/apply-for-postgraduate-teacher-training/blob/master/config/locales/application_states.yml#L193
https://github.com/DFE-Digital/apply-for-postgraduate-teacher-training/blob/master/config/locales/application_states.yml#L233
https://github.com/DFE-Digital/apply-for-postgraduate-teacher-training/blob/master/config/locales/application_states.yml#L300
https://github.com/DFE-Digital/apply-for-postgraduate-teacher-training/blob/master/config/locales/application_states.yml#L310

That will link the emails from this page, that you can use to see when we send what emails:

https://qa.apply-for-teacher-training.education.gov.uk/support/when-emails-are-sent

Reply:
Resolved in f68bd9c